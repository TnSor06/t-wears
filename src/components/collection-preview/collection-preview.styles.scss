.collection-preview {
  display: flex;
  flex-direction: column;
  margin-bottom: 30px;

  .title {
    font-size: 28px;
    margin-bottom: 25px;
  }

  .preview {
    display: flex;
    justify-content: space-between;
  }

  .more {
    color: #357ae8;
    margin-top: 10px;
    font-size: 20px;
    font-weight: 800;
    display: flex;
    justify-content: center;
    &:hover {
      color: #000000;
    }
  }
}
