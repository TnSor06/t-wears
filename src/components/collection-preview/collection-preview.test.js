import React from "react";
import { shallow, mount, render } from "enzyme";
import { CollectionPreview } from "./collection-preview.component";

describe("Collection Preview test", () => {
  const dummyData = {
    title: "Sample data",
    routeName: "sampledata",
    items: [
      {
        id: 1,
        name: "Brown Brim",
        imageUrl: "https://i.ibb.co/ZYW3VTp/brown-brim.png",
        price: 25,
      },
      {
        id: 2,
        name: "Blue Beanie",
        imageUrl: "https://i.ibb.co/ypkgK0X/blue-beanie.png",
        price: 18,
      },
      {
        id: 3,
        name: "Brown Cowboy",
        imageUrl: "https://i.ibb.co/QdJwgmp/brown-cowboy.png",
        price: 35,
      },
      {
        id: 4,
        name: "Grey Brim",
        imageUrl: "https://i.ibb.co/RjBLWxB/grey-brim.png",
        price: 25,
      },
    ],
  };
  // Snapshot testing
  it("Snapshot testing", () => {
    expect(shallow(<CollectionPreview {...dummyData} />)).toMatchSnapshot();
  });
});
