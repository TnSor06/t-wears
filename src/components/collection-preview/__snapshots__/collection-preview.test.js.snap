// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collection Preview test Snapshot testing 1`] = `ShallowWrapper {}`;
